### PR TITLE
Introduce "speed" attribute

### LINKED ISSUE #20: speed variation
hi fmos!
Your plugin is very good for me who create an infinite horizontal slideshow. I just have a question : how can i change the speed of the scrolling? 
perhaps it will be a good thing to add this parameters in the shortcode?

Thx and sorry fo my bad english : i'm just french ;)

### PR DESCRIPTION
**Pull request by artjomsimon**
fixes #20

### CODE REVIEW

File: jquery.photonav.js [hunk 1]
```diff
@@ -177,6 +183,7 @@
 								}
 							} );
 					});
+					console.log("leftEnd: "+leftEnd+" leftStart: "+leftStart+" result: "+Math.abs( leftEnd - leftStart ));
```
Comment (fmos) [inline jquery.photonav.js]: This line seems to be a leftover from your debugging; could you please remove it from the pull request?

File: wp-photonav.php [hunk 2]
```diff
@@ -153,6 +158,4 @@ function get_unique_id() {
 
 if ( class_exists( 'PhotoNav' ) ) {
 	$photonav = new PhotoNav();
-}
-
-?>
+}
```
Comment (fmos) [inline wp-photonav.php]: While you're at it, could you please also add the newline? Thanks!

**Comment by fmos** [comment]
Thanks for this feature implementation. I must admit, I completely forgot about #20  (not closed by me).

**Comment by artjomsimon** [comment]
Ok, thanks for double-checking, that should do it now.
I've still omitted the closing PHP tag since that's the official recommendation by the PHP team, but I'm not too attached to it, just say if you'd like it back.
Thanks!

**Pull request closed without merging**